### DISCUSSION TITLE
32x32px logic components (fix ready)

### DISCUSSION THREAD

**Discussion by TheMasketttaMan** [FakeFishGames/Barotrauma#6543] [Ideas and suggestions]
There is an annoying issue of logic operators not arranging nicely into rows/squares during sub-building due to their sourcerects being some pixels off a nice 32x32 size. Altering these sourcerects & perhaps a sprite atlas is the obvious & lazy way to fix it.

Thankfully, I've already done all the work required in [this mod](https://steamcommunity.com/sharedfiles/filedetails/?id=2590928641). You can find both 6x6 grid sprite atlas and the relevant .xml inside to merge into the main game.

**Comment by Working-Joe** [comment]
This mod is my favourite.

**Comment by esirprus** [comment]
really like it
